src/metrics-modeling/metricbot/extract.py

```
@@ -33,13 +33,17 @@
+GRADING_SCORES_SQL = SQL_DIR / "grading_scores.sql"
 
 
 class ExtractionError(RuntimeError):
     pass
 
 
-def _run_psql(config: Config, sql_file: Path, variables: dict[str, int] | None = None) -> pd.DataFrame:
+def _run_psql(config: Config,
+              sql_file: Path,
+              variables: dict[str, int] | None = None,
+              allow_empty: bool = False) -> pd.DataFrame:
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)